mesonbuild/dependencies/base.py

```diff
@@ -248,7 +248,7 @@ def get_for_machine_from_kwargs(self, kwargs):
         return MachineChoice.BUILD if kwargs.get('native', False) else MachineChoice.HOST
 
 class ExternalDependency(Dependency, HasNativeKwarg):
-    def __init__(self, type_name, environment, language, kwargs):
+    def __init__(self, type_name, environment, kwargs, language: T.Optional[str] = None):
```

Comment:
Does the commentary for the commit containing this change want to say "Dependency constructor signatures" ?